queue/python/queue.py

```diff
@@ -0,0 +1,33 @@
+class Queue:
+    def __init__(self):
+        self.queue = []
+
+    def is_empty(self):
+        return self.queue == []
+
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp67rd4td0/queue/python/queue.py
+++ b/tmp/tmp67rd4td0/queue/python/queue.py
@@ -15,7 +15,7 @@
         return len(self.queue)
 
     def front(self):
-        return self.queue[self.size()-1] 
+        return self.queue[self.size()-1]
 
 
 def main():
```